lib/foreman_tasks/engine.rb

```diff
@@ -57,7 +57,9 @@ class Engine < ::Rails::Engine
 
         security_block :foreman_tasks do |_map|
           permission :view_foreman_tasks, { :'foreman_tasks/tasks' => [:auto_complete_search, :sub_tasks, :index, :show],
-                                            :'foreman_tasks/api/tasks' => [:bulk_search, :show, :index, :summary] }, :resource_type => ForemanTasks::Task.name
+                                            :'foreman_tasks/react' => [:index],
+                                            :'foreman_tasks/api/tasks' => [:bulk_search, :show, :index, :summary],
```

Comment:
Rubocop is not happy about changes in this file

Reply:
Should be fixed now